ReactiveCocoa/UIKit/iOS/UISearchBar.swift

```diff
@@ -41,4 +45,10 @@ extension Reactive where Base: UISearchBar {
 			.map { [unowned base] in base.text }
 	}
 	
+	/// A void signal emitted by the search bar upon any click on the cancel button
+	public var cancelButtonClicked: Signal<Void, NoError> {
+		return proxy.intercept(#selector(UISearchBarDelegate.searchBarCancelButtonClicked))
+			.map { () -> Void in }
```

Comment:
This `map` isn't needed. `Void` and `()` are synonymous, so this isn't actually changing the value. Can you remove it?